### ISSUE TITLE
t507-lpddr4

### ISSUE THREAD

**Issue by ua1arn** [iuncuim/u-boot#1]
Hello.
What a model of board used by you for testings t507-lpddr4 modifications?
Have you any success? I am working on bare-metal for HelperBoard T507 - and unsuccesful...

```
testing 32-bit width, rank = 2
read training failed!
testing 32-bit width, rank = 1
read training failed!
testing 16-bit width, rank = 2
read training failed!
testing 16-bit width, rank = 1
read training failed!
This DRAM setup is currently not supported

```
Version for real H616 equipped by DDR3 (Orange Pi Zero 2) work fine.

**Comment by iuncuim** [comment]
yes, tested with myir yt507 and 9tripod x507cb1. You should find dram_para from vendor image. It can be find in vendor bsp (sys_config.fex) or in dtb.
After that put necessary values to defconfig.

**Comment by iuncuim** [comment]
and try h616-dram-lpddr3 branch.
Because t507-lpddr4 was first attempt with many hardcoded values.
